python_tools/check_db.py

```diff
@@ -58,21 +58,28 @@ def check_files(folder_path: str) -> bool:
 def assign_dataclass(data: list) -> Result:
     """ Assigns the data to the Result dataclass """
     try:
+        def to_float(value):
```

Comment:
Yeah this means we'll end up passing bad values into the database. I'll tag a few errors that should be thrown.

You've basically found a bug in the AUS scraping tool and addressed it wrongly.